examples/gno.land/p/samourai/daocond/cond_govdao.gno

```diff
@@ -0,0 +1,170 @@
+package daocond
+
+import (
+	"errors"
+	"math"
+	"strconv"
+	"strings"
+
+	"gno.land/p/demo/json"
+	"gno.land/p/demo/ufmt"
+)
+
+var roleWeights = []float64{3.0, 2.0, 1.0}
+
+func GovDaoCondThreshold(threshold float64, roles []string, hasRoleFn func(memberId string, role string) bool, usersWithRoleCountFn func(role string) uint32) Condition {
```

Comment:
should we remove this or rename to gnolove?

Reply:
[9b3e9fd](https://github.com/samouraiworld/gno/pull/1/commits/9b3e9fd89a1d65f461a557352b2dc271142a4f0c)